test/inductor/test_torchinductor.py

```diff
@@ -604,6 +604,54 @@ def fn(x, y, z):
 
         self.common(fn, (x, y, z))
 
+    # lowering operation tests
+    def test_add_uint8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([128], dtype=torch.uint8)
+
+        self.common(fn, x)
+
+    def test_add_int8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([120], dtype=torch.int8)
+
+        self.common(fn, x)
+
+    def test_add_int16_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int32)
+
+        x = torch.tensor([35000], dtype=torch.int16)
+
+        self.common(fn, x)
+
+    def test_multiply_uint8_tensor(self):
+        def fn(x):
+            return (x + x).to(torch.int16)
+
+        x = torch.tensor([8], dtype=torch.uint8)
```

Comment:
```suggestion
        x = torch.tensor([100], dtype=torch.uint8)
```

Reply:
What is the thinking in changing the number here. I was trying to get it to 256 which is 0. I could add anotehr test with 100 to check a non zero the over flow case. Or have I misunderstood?